recompose-flow.md

```diff
@@ -1,19 +1,19 @@
-# Flow support in recompose
+# Flow support in Recompose
 
 ## History
 
-One month ago official flow library definition was landed in recompose.
-It was a long way before we added it having that [PR](https://github.com/acdlite/recompose/pull/241) was added by @GiulioCanti one year ago.
+One month ago the official Flow library definition was landed in Recompose.
+It was a long time coming, considering the original [PR](https://github.com/acdlite/recompose/pull/241) was created by @GiulioCanti a year ago.
 
-There was one problem, I got stuck with one of simplest enhancers in recompose library `withProps`.
+There was one problem. I got stuck on one of simplest enhancers in the Recompose library: `withProps`.
 
-Type inference had worked but not in a way I expected, errors were not detected or were cryptic, readability of intersection types also didn't make me happy.
-The only way was to declare almost every input-output type for each enhancer but I'm too lazy to do it well.
-Playing with type definitions didn't give me a better result so I gave up.
+Type inference worked, but not in a way I expected; errors were not detected or error messages were cryptic, and the readability of intersection types didn't make me happy.
+The only fix was to declare almost every input--output type for each enhancer --- but I'm too lazy to do that well.
+Playing with type definitions didn't give me a better result, so I gave up.
 
-The game changer was object type spread landed in [flow v0.42.0](https://github.com/facebook/flow/releases/tag/v0.42.0)
+The game changer was object type spread, which landed in [Flow v0.42.0](https://github.com/facebook/flow/releases/tag/v0.42.0).
```

Comment:
Will everyone in your audience understand immediately what "object type spread" is? Or would it be helpful to add a little description here, like "The game changer was object type spread, which allows you to spread one object type into another. It landed in..."